src/test/resources/selenide-test.css

```
@@ -0,0 +1,123 @@
+/* Shared cosmetic styling for Selenide test fixture HTML pages.
+   Applied via <link rel="stylesheet" href="/selenide-test.css">.
+   The page banner is a <header class="selenide-test-banner"> so it does not
+   collide with tests using tagName("div") / $$("div") selectors. */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
```

Stylelint (17.10.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap'" (import-notation)

(import-notation)